src/routes/error-handlers.ts

```diff
@@ -0,0 +1,53 @@
+import { Request, Response, NextFunction } from "express";
+import { StatusCodes } from "http-status-codes";
+import { EasyRoutingConfig, ErrorResponse, HttpError } from "../core/models";
+
+/**
+ * Default error handler
+ *
+ * Send an error response with status code from error or 500 is none is provided
```

Comment:
Change to:
_Send an error response with status code from error, or 500 if none is provided_